packages/governance/src/binaryVoteCounter.js

```diff
@@ -1,5 +1,7 @@
 import { makePromiseKit } from '@endo/promise-kit';
-import { makeExo, keyEQ, makeScalarMapStore } from '@agoric/store';
+import { keyEQ } from '@endo/patterns';
+import { makeExo } from '@endo/exo';
+import { makeScalarMapStore } from '@agoric/vat-data';
```

Comment:
```suggestion
import { makeScalarMapStore } from '@agoric/store';
```
IIUC the purpose of this PR, I'd expect you to prefer to import `makeScalarMapStore` from `@agoric/store`, where it originates, rather than `@agoric/vat-data` which imports and re-exports it. This seems more consistent with your preference (which I share) to import, for example, `makeExo` from `@endo/exo` where it originates, rather than `@agoric/store`, which imports and re-exports it.

Likewise for other similar changes from `@agoric/store` to `@agoric/vat-data` below.

FWIW, I plan to migrate `@agoric/store` to become `@endo/store`, and to move most of `@agoric/zone` to `@endo/zone`. Longer term, I expect such manual use of `makeExo` and `makeScalarMapStore` to migrate to using the corresponding Zone APIs. Just FYI; doesn't affect this PR.

Reply:
Agreed. I mistakenly had in mind that `vat-data` sourced the functionality from `globalThis` and thus didn't entail a package dependency on `@agoric/store`. 

I'll revert the changes that preferred vat-data.